sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -31,6 +31,7 @@
 import com.google.common.collect.Iterators;
 import com.google.common.collect.Lists;
 import com.google.common.io.Closeables;
+
```

Comment:
I usually take care not to impose my code standards in older code it am modifying it anyway. But ok :-).